w3af/plugins/crawl/url_payment_finder.py

```diff
@@ -0,0 +1,219 @@
+"""
+url_payment_finder.py
```

Comment:
Maybe rename it to `payment_webhook_finder.py`?

All crawl plugins find urls, so it is a little bit too generic to use `url_...` naming.

Reply:
Remember to rename the filename, this comment content and class name.

Reply:
Thank you for the detailed description for the requested changes. It's really helpfull.
I started with the simple changes like this one. I will try make the other changes before next week so I may come back with some questions. I will comment "done" for the ones I've done already.